sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizeHealthcareEntitiesResultCollection.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Azure.AI.TextAnalytics.Models;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Collection of <see cref="RecognizeHealthcareEntititesResult"/> objects corresponding
```

Comment:
according to the implementation it should be 
```suggestion
    /// Collection of <see cref="DocumentHealthcareResult"/> objects corresponding
```